### DISCUSSION TITLE
PCL 启动器不能登录部分其他启动器可以登录的皮肤站

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#5135] [讨论]
Ely.by

**Comment by MoYuan-CN** [comment]
我并不能理解什么是 Ely.by，尝试作为网站打开也失败了
需要更多信息及详细日志……

**Reply by hejiehao** [reply]
![image](https://github.com/user-attachments/assets/b187ef92-ecbc-48f5-b0ef-4ca6147426e3)

**Comment by LinQingYuu** [comment]
>Ely.by

根据调查，此皮肤站似乎没有按照[规范](https://github.com/yushijinhun/authlib-injector/wiki/Yggdrasil-%E6%9C%8D%E5%8A%A1%E7%AB%AF%E6%8A%80%E6%9C%AF%E8%A7%84%E8%8C%83)编写
规范中要求的登录地址是 `/authserver/authenticate`
所以 PCL 会给验证地址叠加一个 `/authserver/authenticate` 字符串
但是文档中给出的的登录地址是 `/auth/authenticate`
不过网站提供了 `X-Authlib-Injector-API-Location` PCL 没有读取，算是一个 Bug

**Reply by allMagicNB** [reply]
你看它返回 404 了？

**Comment by LTCatt** [comment]
转 #5168

**Comment by allMagicNB** [comment]
？

**Comment by allMagicNB** [comment]
我的问题没有解决。

**Reply by LTCatt** [reply]
如果有不对的转正式 issue 里发啊（

**Reply by allMagicNB** [reply]
正式的 Issue 不是在指这个问题。

**Reply by allMagicNB** [reply]
elyby/accounts#35
